javav2/usecases/creating_photo_analyzer_async/Readme.md

```diff
@@ -0,0 +1,1532 @@
+        msg.addBodyPart(att);
+
+        // Try to send the email
+        try {
+            System.out.println("Attempting to send an email through Amazon SES " + "using the AWS SDK for Java...");
+
+            Region region = Region.US_WEST_2;
+            SesClient client = SesClient.builder()
+                    .credentialsProvider(EnvironmentVariableCredentialsProvider.create())
+                    .region(region)
+                    .build();
+
+            ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+            message.writeTo(outputStream);
+            ByteBuffer buf = ByteBuffer.wrap(outputStream.toByteArray());
+            byte[] arr = new byte[buf.remaining()];
+            buf.get(arr);
+
+            SdkBytes data = SdkBytes.fromByteArray(arr);
+            RawMessage rawMessage = RawMessage.builder()
+                    .data(data)
+                    .build();
+
+            SendRawEmailRequest rawEmailRequest = SendRawEmailRequest.builder()
+                    .rawMessage(rawMessage)
+                    .build();
+
+            client.sendRawEmail(rawEmailRequest);
+
+        } catch (SesException e) {
+            System.err.println(e.awsErrorDetails().errorMessage());
+            System.exit(1);
+        }
+        System.out.println("Email sent with attachment.");
+        }
+       }
+```
+
+ ### WorkItem class
+
+ The following Java code represents the **WorkItem** class.
+
+```java
+     package com.example.photo;
+
+    public class WorkItem {
+
+     private String key;
+     private String name;
+     private String confidence ;
+
+     public void setKey (String key) {
+        this.key = key;
+     }
+
+     public String getKey() {
+        return this.key;
+     }
+
+     public void setName (String name) {
+        this.name = name;
+     }
+
+     public String getName() {
+        return this.name;
+     }
+
+     public void setConfidence (String confidence) {
+        this.confidence = confidence;
+     }
+
+     public String getConfidence() {
+        return this.confidence;
+      }
+     }
+```
+
+### WriteExcel class
+
+The following Java code represents the **WriteExcel** class.
+
+```java
+    package com.example.photo;
+
+    import jxl.CellView;
+    import jxl.Workbook;
+    import jxl.WorkbookSettings;
+    import jxl.format.UnderlineStyle;
+    import jxl.write.Label;
+    import jxl.write.Number;
+    import jxl.write.WritableCellFormat;
+    import jxl.write.WritableFont;
+    import jxl.write.WritableSheet;
+    import jxl.write.WritableWorkbook;
+    import jxl.write.WriteException;
+    import org.springframework.stereotype.Component;
+    import java.io.IOException;
+    import java.util.List;
+    import java.util.Locale;
+
+    @Component
+    public class WriteExcel {
+
+     private WritableCellFormat timesBoldUnderline;
+     private WritableCellFormat times;
+
+     // Returns an InputStream that represents the Excel report
```

Comment:
Once again, check for periods at the end of comment sentences throughout.

Reply:
fixed